src/basic/rlimit-util.c

```diff
@@ -76,6 +80,42 @@ int setrlimit_closest_all(const struct rlimit *const *rlim, int *which_failed) {
         return 0;
 }
 
+int setpriority_closest(int nice) {
```

Comment:
nice is also a symbol exposed by glibc. I think it would be nice to avoid the naming clash here, even though it's not technically wrong

Reply:
No biggie, but we have a check in the build system to only enable `-Wshadow` if it does **not** diagnose this a failure. I think it OK, there's always going to be conflicts with non-namespaced functions, and there is no ambiguity for a local var.

Reply:
well, but in this case the function nice() and the variable actually even are about the same concepts, which makes me really want to avoid any ambiguity here...